common/i18n/resources/en.js

```diff
@@ -117,6 +117,11 @@ module.exports = {
         ascending: 'ascending',
         descending: 'descending',
       },
+      code: 'code',
+      label: 'label',
+      scientific_name: 'Scientific name',
+      file_uuid: 'File uuid',
+      file_name: 'File name',
```

Comment:
why not to add those to surveyForm where all nodeDefs labels are?

Reply:
Ok, added a dataVisHeader section in nodeDefs